painters/submission_jxs.py

```diff
@@ -0,0 +1,117 @@
+from os.path import join
+
+import numpy as np
+from data_provider import SUBMISSION_INFO_FILE, DATA_DIR
+from data_provider import load_organized_data_info
+from utils import append_to_file
+from utils import read_lines_in_batches
+from keras.models import model_from_json
+from os.path import basename
+
+IMGS_DIM_1D = 256
+SUBMISSION_FILE = join(DATA_DIR, 'submission.csv')
+BATCH_SIZE = 10000
+FILES_TO_AVG = {}
+
+
+def _create_submission_file_avg_cnns():
+    data_info = load_organized_data_info(IMGS_DIM_1D)
+    model = _get_model()
+    X_test, names = _average_embedded_test_data(model, data_info)
+    features_lookup = {n: f for n, f in zip(names, X_test)}
+    _create_submission_file(
+        BATCH_SIZE, features_lookup, _calculate_batch_prediction_dot)
+
+
+def _get_model():
+    with open('../models/model.json', 'r') as json_file:
+        model = model_from_json(json_file.read())
+    model.load_weights('../models/weights.h5')
```

Comment:
This will be a little cleaner to make the model path `../models` an argument to this function.

Reply:
When operating on filepaths in Python, prefer using `os.path` -- e.g. `os.path.join("..", "models", "weights.h5")`